bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+   2.8. [What is BLS Voting](#what-is-bls-voting) \
+   2.9. [Create BLS Key](#create-bls-key)
+3. [Run Node](#run-node)
+4. [Create Validator](#create-validator)
+5. [Verify your Validator](#verify-your-validator)
+
+## Prerequisites
+
+This system spec has been tested by validators and found to be comfortable:
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+You can run Babylon on lower-spec hardware for each component, but you may find that it is not highly performant or prone to crashing.
+## Setup
+
+We need to setup the following before we can classify as a validator. 
+- Setup node and configure
+- Create keyring
+- Get funds
+- Create BLS Key
+- Run node
+
+### Setup Node and Configure
+
+<!-- TODO: check add in the correct tag for the testnet -->
+#### Build node from `x` tag. 
+
+Here we will checkout the tag in which we will use. 
+
+` git checkout x`
+
+#### Initialize the chain
+
+Run  `babylond init --chain-id bbn-test-5` in the terminal 
+
+It should generate all of the necessary files such as `app.toml`, `client.toml`, `genesis.json`
+
+#### Update the configuration
+
+We need to then change the configuration details in the following files. 
+- In `app.toml` under `[btc-network]`change network to `signet` and `iavl-cache-size = 0`
+- In the `config.toml` add in the `seeds` and `persistent-peers` token (this can be the same token)
+
+#### Retrieve the genesis file 
+
+We next want to add in the genesis file from thee network. How we can do this is either copy the genesis file from  `https://rpc.devnet.babylonlabs.io/genesis` or we can use the terminal.
+
+```
+wget https://github.com/babylonlabs-io/networks/raw/main/bbn-test-5/genesis.tar.bz2  
+tar -xjf genesis.tar.bz2 && rm genesis.tar.bz2  
+mv genesis.json ~/.babylond/config/genesis.json
+```
+
+This file needs to overwrite the existing genesis file in the `~/.babylond/config/genesis.json` and please ensure that the `chain-id` is correct.
+
+#### Create a Keyring
+
+What we want to do next is create a key we can do this by running:
+
+`babylond --keyring-backend test keys add <name>`
+
+We use `--keyring-backend test`, which specifies which backend to use for the keyring, `test` stores keys unencrypted on disk. At the end we add a `name` which is the name in which we will give to the key. 
+
+What should return is a message that the key was added successfully. An example of what this should look like is below:
+
+```
+- address: bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9
+  name: test-val
+  pubkey: '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"Ayau+8f945c1iQp9tfTVaCT5lzhD8n4MRkZNqpoL6Tpo"}'
+  type: local
+```
+
+#### Get Funds
+<!-- This needs to be updated to the correct testnet and potentially need help with the correct instructions as I cant access the faucet -->
+Request Funds from the Babylon Testnet Faucet​ [here](https://faucet.devnet.babylonlabs.io/) 
+
+##### What is BLS Voting
```

Comment:
We can provide a brief explanation of BLS voting here and work on developing a separate technical document explaining the need for BLS. We can link to this document from here. Let's add a task on developing this document.